test-case-reporting/app.ts

```diff
@@ -67,6 +67,24 @@ function extractPageInfo(req: express.Request): PageInfo {
   };
 }
 
+// We need the testRun index in mustache to give each collapsable
+// its own id.
+function enumerateTestRun(
+  testRun: TestRun,
+  index: number
+): {testRun: TestRun; index: number} {
+  return {testRun, index};
+}
+
+function lowerCaseStatus({status, ...rest}: TestRun): any {
```

Comment:
At some point I think I'd prefer to see a single helper with something like `testRunsForView` that takes a list of test runs and does everything, including indexes, capitalization, and possible eventually finding build IDs or constructing other URLs. Fine with punting on it for now

Reply:
I like that; agreed on punting it, though.